resources/build/version/src/fixupHistory.ts

```diff
@@ -197,7 +201,30 @@ const splicePullsIntoHistory = async (pulls: PRInformation[]): Promise<number> =
     writeFileSync('HISTORY.md', newHistory, 'utf8');
   }
 
-  return historyChunks.current.length - 3; // - 3 for header + blanks
+  return {count: historyChunks.current.length - 3, pulls: currentPulls}; // - 3 for header + blanks
```

Comment:
Really, we could just return `pulls` and use `pulls.length`! But I leave that for another day?